pkgs/os-specific/linux/multipath-tools/default.nix

```
@@ -16,7 +16,13 @@ stdenv.mkDerivation rec {
   ];
 
   postPatch = ''
-    substituteInPlace libmultipath/Makefile --replace /usr/include/libdevmapper.h ${lib.getDev lvm2}/include/libdevmapper.h
+    substituteInPlace libmultipath/Makefile \
+      --replace /usr/include/libdevmapper.h ${lib.getDev lvm2}/include/libdevmapper.h
+
+    substituteInPlace multipathd/multipathd.service \
+      --replace /sbin/modprobe ${lib.getBin kmod}/sbin/modprobe \
```

```suggestion
      --replace /sbin/modprobe ${lib.getBin kmod}/bin/modprobe \
```
NixOS has no sbin.

``/nix/store/5mrjf49b5ys5gmigxiwar40qdp8wixk8-kmod-27/bin/modprobe``

---

We're both right!

```
$ which kmod 
/nix/store/38gy8j3x4dyfgjmsvwfdb1c2k497s4qz-kmod-27/bin/kmod

$ /nix/store/38gy8j3x4dyfgjmsvwfdb1c2k497s4qz-kmod-27/
bin/     include/ lib/     sbin/    share/   

$ /nix/store/38gy8j3x4dyfgjmsvwfdb1c2k497s4qz-kmod-27/sbin/
depmod    insmod    kmod      lsmod     modinfo   modprobe  rmmod     
```


https://github.com/NixOS/nixpkgs/blob/802cc885e4020eecf3e282be02dd8307d2726352/pkgs/os-specific/linux/systemd/default.nix#L425-L431